frontend/src/metabase/app.js

```diff
@@ -135,6 +143,29 @@ angular.module('metabase', [
     $routeProvider.when('/admin/people/', route);
     $routeProvider.when('/admin/settings/', { ...route, template: '<div class="full-height" mb-redux-component />' });
 
+    $routeProvider.when('/reference', route);
```

Comment:
I gave this a try just now and it seems to be causing some weird problems.

Basically once I remove those routes, every time I navigate to a new route I get this in the console:

```
angular.js?d2dc:10126 TypeError: Cannot read property 'controller' of undefined
    at eval (eval at <anonymous> (app.hot.bundle.js?c0dfc21…:1), <anonymous>:360:35)
    at Scope.$broadcast (eval at <anonymous> (vendor.hot.bundle.js?c0dfc21…:1), <anonymous>:13093:28)
    at afterLocationChange (eval at <anonymous> (vendor.hot.bundle.js?c0dfc21…:1), <anonymous>:9972:18)
    at eval (eval at <anonymous> (vendor.hot.bundle.js?c0dfc21…:1), <anonymous>:9960:13)
    at Scope.$eval (eval at <anonymous> (vendor.hot.bundle.js?c0dfc21…:1), <anonymous>:12811:28)
    at Scope.$digest (eval at <anonymous> (vendor.hot.bundle.js?c0dfc21…:1), <anonymous>:12623:31)
    at Scope.$apply (eval at <anonymous> (vendor.hot.bundle.js?c0dfc21…:1), <anonymous>:12915:24)
    at HTMLHtmlElement.eval (eval at <anonymous> (vendor.hot.bundle.js?c0dfc21…:1), <anonymous>:9913:24)
    at eval (eval at <anonymous> (vendor.hot.bundle.js?c0dfc21…:1), <anonymous>:2853:10)
    at forEach (eval at <anonymous> (vendor.hot.bundle.js?c0dfc21…:1), <anonymous>:325:18)(anonymous function) @ angular.js?d2dc:10126(anonymous function) @ angular.js?d2dc:7398Scope.$broadcast @ angular.js?d2dc:13095afterLocationChange @ angular.js?d2dc:9972(anonymous function) @ angular.js?d2dc:9960Scope.$eval @ angular.js?d2dc:12811Scope.$digest @ angular.js?d2dc:12623Scope.$apply @ angular.js?d2dc:12915(anonymous function) @ angular.js?d2dc:9913(anonymous function) @ angular.js?d2dc:2853forEach @ angular.js?d2dc:325eventHandler @ angular.js?d2dc:2852
VM33470:267 
creating store
```

And it seems to be reloading data on every nav (possibly because it's recreating the store on each route? see last line in console output).